action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
       ideas.
     default: stable-2.11
     required: true
+  docker-image:
+    description: Docker image used by ansible-test
+    default:
```

Comment:
Should this be
```suggestion
    default: default
```
?

Reply:
Actually there is no target specified for `ansible-test` and it works well.